shared/common-adapters/relative-popup-hoc.desktop.js

```diff
@@ -203,8 +207,10 @@ function ModalPositionRelative<PP>(
       const style = collapseStyles([
         computePopupStyle(this.props.position, targetRect, popupNode.getBoundingClientRect()),
         this.props.style,
+        need2ndCall ? {opacity: 0} : {},
       ])
       this.setState({style})
+      need2ndCall && setTimeout(() => this._computeStyle(this.props.targetRect, true))
```

Comment:
This is kind of hacky, but is the only way I could make it work.

Reply:
lets discuss, you can likely use componentDidUpdate or getSnapshotBeforeUpdate

Reply:
This is already called in `componentDidUpdate`. I think it has to do with browser figuring out the actual width. I'll see if I can make `getSnapshotBeforeUpdate` work.

Reply:
It works!! Thanks